rllib/agents/pg/pg.py

```diff
@@ -1,9 +1,32 @@
+"""
+Implementation of a vanilla policy gradients algorithm. Based on:
+
+[1] - Policy Gradient Methods for Reinforcement Learning with Function
+    Approximation.
+    Sutton, McAllester, Singh, Mansour - AT&T Labs - 1999
+    https://papers.nips.cc/paper/1713-policy-gradient-methods-for-
+    reinforcement-learning-with-function-approximation.pdf
+[2] - Simple Statistical Gradient-Following Algorithms for Connectionist
+    Reinforcement Learning.
+    Williams - College of Computer Science - Northeastern University - 1992
+    http://www-anw.cs.umass.edu/~barto/courses/cs687/williams92simple.pdf
```

Comment:
Should we add "This file defines the distributed trainer for policy gradients; see pg_policy.py for the definition of the policy loss?"

Reply:
done

Reply:
done